_data/sites.json

```diff
@@ -7427,6 +7447,17 @@
         ]
     },
 
+    {
+        "name": "Letudiant",
+        "url": "https://my.letudiant.fr/tableau-de-bord/parametres/",
```

Comment:
This is giving page not found error, does it only work when logged in?

Reply:
Ops!! In fact, the link was wrong, but I've just corrected this.